frontend/src/pages/Traces/TraceFlameGraphNode.tsx

```diff
@@ -46,10 +86,21 @@ export const TraceFlameGraphNode = memo<Props>(
 			? depth * (lineHeight + 3) + (ticksHeight + outsidePadding)
 			: ticksHeight + outsidePadding
 		const isSelectedSpan = selectedSpan?.spanID === span.spanID
-		const error = errors.find((error) => error.span_id === span.spanID)
-		const fill = isSelectedSpan ? '#744ed4' : '#e7defc'
-		const color = isSelectedSpan ? '#fff' : '#744ed4'
-		const stroke = error ? '#f00' : '#f9f8f9'
+		const hasError = errors.find((error) => error.span_id === span.spanID)
+		const isDbSpan = !!span.traceAttributes?.db?.system
+		const isFrontendSpan =
+			span.traceAttributes?.highlight?.type === 'http.request'
+		const themeName = isDbSpan
+			? 'yellow'
+			: isFrontendSpan
+			? 'green'
+			: hasError
+			? 'red'
+			: 'purple'
+		const theme = spanThemes[themeName] ?? spanThemes['purple']
+		const fill = isSelectedSpan ? theme.selectedBackend : theme.background
+		const color = isSelectedSpan ? theme.selectedColor : theme.color
+		const stroke = isSelectedSpan ? theme.selectedBackend : theme.border
```

Comment:
Will need to follow up on how we colorize these spans. Will evolve this in future PRs.